app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/image_import_workflow.rb

```diff
@@ -0,0 +1,111 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::ImageImportWorkflow < ManageIQ::Providers::AnsiblePlaybookWorkflow
+  def load_transitions
+    self.state ||= 'initialize'
+
+    {
+      :initializing      => {'initialize'         => 'waiting_to_start'},
+      :start             => {'waiting_to_start'   => 'pre_execute'},
+      :pre_execute       => {'pre_execute'        => 'execute'},
+      :execute           => {'execute'            => 'running'},
+      :poll_runner       => {'running'            => 'running'},
+      :post_execute      => {'running'            => 'post_execute_poll'},
+      :post_execute_poll => {'post_execute_poll'  => 'post_execute_poll'},
+      :finish            => {'*'                  => 'finished'},
+      :abort_job         => {'*'                  => 'aborting'},
+      :cancel            => {'*'                  => 'canceling'},
+      :error             => {'*'                  => '*'}
+    }
```

Comment:
Since most of this state transition is the same as the one from the base class this method could be simplified to
```
def load_transitions
  super.merge(:post_execute => {'running' => 'post_execute_poll'}, :post_execute_poll => {'post_execute_poll' => 'post_execute_poll'})
end
```

So that you don't overwrite the whole set of state transitions.  This means less likely to break if we change something in core

Reply:
Got it, I will change now.